modules/ContextUtils.js

```diff
@@ -43,8 +43,9 @@ export function ContextProvider(name) {
     },
 
     componentDidUpdate() {
+      const nextValue = this.getChildContext()[name]
```

Comment:
This is safe because React makes no guarantees about when this method can be called